addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -40,19 +40,22 @@
                  <span t-esc='props.formatCurrency(props.data.amount_total + props.data.rounding_applied)' class="pos-receipt-right-align"/>
               </div>
             </t>
-            <br/><br/>
 
             <!-- Payment Lines -->
 
-            <div class="paymentlines" t-foreach="props.data.paymentlines" t-as="line" t-key="line_index">
-                <t t-esc="line.name" />
-                <span t-esc="props.formatCurrency(line.amount, false)" class="pos-receipt-right-align"/>
+            <div class="pos-receipt-pay">
+                <div class="paymentlines" t-foreach="props.data.paymentlines" t-as="line" t-key="line_index">
+                    <t t-esc="line.name" />
+                    <span t-esc="props.formatCurrency(line.amount, false)" class="pos-receipt-right-align"/>
+                </div>
             </div>
 
-            <div class="pos-receipt-amount receipt-change mt-2">
-                CHANGE
-                <span t-esc="props.formatCurrency(props.data.change)" class="pos-receipt-right-align"/>
-            </div>
+            <t t-if="props.data.change != 0">
+                <div class="pos-receipt-amount receipt-change mt-2">
```

Comment:
you can put the `t-if` on the `div` element ( this is more a thing of personal preference, so you are free to do as you see fit )

Reply:
I wanted to but it didn't work 🤷‍♀️

Reply:
if you just do ```xml
<div t-if="props.data.change != 0" class="pos-receipt-amount receipt-change mt-2">
```
it doesn't work?

Reply:
yes it doesn't